docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -3333,6 +3333,12 @@ Start the sensor:
 
 The DHCP traffic should now be reflected on your PacketFence server.
 
+Switch login access
+~~~~~~~~~~~~~~~~~~~
+
+PacketFence is able to act as an authentication and authorization service for granting command-line interface (CLI) access to switches.
+PacketFence currently supports Cisco switches and these must be configured using the following guide: http://www.cisco.com/c/en/us/support/docs/security-vpn/remote-authentication-dial-user-service-radius/116291-configure-freeradius-00.html
+From the PacketFence's web admin interface, you must configure an Admin Access role that contains the action SWITCH_LOGIN_READ or SWITCH_LOGIN_WRITE and assign this role to an internal user or in an Administration rule in an internal source.
```

Comment:
Also can you put the location of the admin roles configuration : 

you must configure an Admin Access role (Configuration->Admin access)